.golangci.yaml

```diff
@@ -38,7 +38,7 @@ linters-settings:
   dupl:
     threshold: 100
   funlen:
-    lines: 100
+    lines: 120
```

Comment:
Why do we increase these values instead of using `// nolint: funlen` or `// nolint: gocylco` to override the violating instances? 

The default values are reasonable, and would be able to identify the outliers by gripping on `// nolint`

Reply:
There is only one function that is an outlier
```
// nolint: funlen, gocyclo
func (cmd *ConfigureCmd) Run(c *cc.CommonCtx) error {
```